common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -198,6 +199,22 @@ def test_parse_youtube_empty(self):
              '1.50': ''}
         )
 
+    @patch.object(settings, 'FEATURES', create=True, new={})
+    @patch.object(settings, 'COMPLETION_VIDEO_COMPLETE_PERCENTAGE', create=True, new=None)
+    @patch.object(settings, 'YOUTUBE', create=True,
+                  new={'TEST_TIMEOUT': 1500,
+                       'API': 'http://127.0.0.1/get_youtube_api',
+                       'METADATA_URL': 'http://127.0.0.1/test_youtube/'})
+    @patch('xmodule.video_module.transcripts_utils.get_transcripts_from_youtube')
+    def test_student_public_view(self, mock_transcripts_from_youtube):
```

Comment:
A better place for testing this would be in this class: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/tests/test_video_mongo.py#L227.

Reply:
Moved it. Thanks.